app/(auth)/new-password/layout.tsx

```
@@ -0,0 +1,15 @@
+
+export default function ForgotPasswordLayout(
+    {
+        children
+    }: Readonly<{
+        children: React.ReactNode;
+    }>
+){
+    <>
+    {children}
+    </>
+}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)